CHANGELOG.rst

```diff
@@ -1,5 +1,5 @@
-sunpy 2.1.0 (2020-02-21)
-========================
+2.1.0 (2020-02-21)
```

Comment:
When the next major release of towncrier comes out we can change the template so we don't need to edit the generated title.